docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '2'
+services:
+  calypso:
+    image: node:6.10.0
+    extra_hosts:
+      - 'calypso.localhost:0.0.0.0'
+    environment:
+      CALYPSO_ENV: 'development'
+      NODE_ENV: 'development'
+      HOST: 'calypso.localhost'
+      PORT: 3000
+    ports:
+      - '3000:3000'
+    volumes:
+      - ./:/calypso
+      - node_modules_volume:/calypso/node_modules
+      - hooks_volume:/calypso/.git/hooks
+    working_dir: /calypso
+    command: make run
```

Comment:
I'm not sure but I don't think we can enable signal forwarding this way. `command: ["make", "run"]` might help. 

Without signal forwarding, when you write `docker-compose down`, `SIGTERM` will not be delivered to the right process (make) and the devserver, compose will wait 10 seconds and force kill.